src/popup/stores/SessionStore.ts

```diff
@@ -13,16 +14,27 @@ const INIT_VALUES = {
 
 export default class SessionStore {
   @observable public networkIndex: number = INIT_VALUES.networkIndex;
+  @observable public networks: QryNetwork[] = [];
   @observable public loggedInAccountName?: string = INIT_VALUES.loggedInAccountName;
   @observable public info?: Insight.IGetInfo = INIT_VALUES.info;
   @computed public get qtumBalanceUSD() {
     return isUndefined(this.qtumUSD) ? 'Loading...' : `$${this.qtumUSD} USD`;
   }
+  @computed public get networkName() {
+    return this.networks[this.networkIndex].name;
+  }
+  @computed public get isMainNet() {
+    return this.networkName === NETWORK_NAMES.MAINNET;
+  }
+  @computed public get networkMarkup() {
```

Comment:
nit: networkValueDesc(). markup doesnt seem intuitive of what its being used for.